# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esnacc/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- SNACC/snacc.h.vanilla	2008-08-25 07:31:25.000000000 +0200
+++ SNACC/snacc.h	2008-08-25 07:31:37.000000000 +0200
@@ -125,32 +125,12 @@
 
 #endif /* __cplusplus */
 
-/*
- *  Inspired by gdb 4.0, for better or worse...
- *  (grabbed from Barry Brachman - MS)
- *
- *  These macros munge C routine declarations such
- *  that they work for ANSI or non-ANSI C compilers
- */
-#ifdef __USE_ANSI_C__
-
 #define PROTO( X)			X
 #define PARAMS( arglist, args)  	(args)
 #define NOPARAMS()	        	(void)
 #define _AND_				,
 #define DOTS				, ...
 
-#else /* !__USE_ANSI_C__ */
-
-#define PROTO( X)			()
-#define PARAMS( arglist, args)	 	arglist args;
-#define NOPARAMS()	        	()
-#define _AND_				;
-#define DOTS
-#define void                            char
-
-#endif /* __USE_ANSI_C__ */
-
 #include "policy.h"
 
 #if COMPILER
--- SNACC/c++-lib/src/snaccexcept.cpp.vanilla	2008-08-26 09:57:00.000000000 +0200
+++ SNACC/c++-lib/src/snaccexcept.cpp	2008-08-26 09:57:05.000000000 +0200
@@ -2,6 +2,7 @@
 //
 //
 #include "snaccexcept.h"
+#include <string.h>
 
 
 namespace SNACC 
--- SNACC/c++-lib/src/asn-rvsbuf.cpp.vanilla	2008-08-26 09:56:12.000000000 +0200
+++ SNACC/c++-lib/src/asn-rvsbuf.cpp	2008-08-26 09:56:17.000000000 +0200
@@ -1,4 +1,5 @@
 #include "asn-buf.h"
+#include <string.h>
 
 using namespace SNACC;
 
--- SNACC/c++-lib/src/asn-fileseg.cpp.vanilla	2008-08-26 09:56:38.000000000 +0200
+++ SNACC/c++-lib/src/asn-fileseg.cpp	2008-08-26 09:56:42.000000000 +0200
@@ -1,5 +1,6 @@
 #include "asn-buf.h"
 #include "snaccexcept.h"
+#include <string.h>
 
 using namespace SNACC;
 
